### PR TITLE
Governance alarm 

### PR DESCRIPTION
**Pull request by skonhwang**

### CODE REVIEW

File: plugins/node_governance_alarm/main.go [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+		needToVote += voteMsg + "\n"
+		log.Info().Str("module", "plugin").Msg(voteMsg)
+	}
+	return needToVote
+}
+
+func findLastestProposalId() {
+	proposalId = 0
+	cntNotFoundProp := 0
+	fmt.Println("zz", proposalId)
```
[the remark this answers is not in the document]
Comment (skonhwang) [reply plugins/node_governance_alarm/main.go]: @gnongs Thanks bro~

File: rpc/cosmos/governance.go [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+		Options    []struct {
+			Option string `json:"option"`
+			Weight string `json:"weight"`
+		} `json:"options"`
+	} `json:"vote"`
+}
+
+func GetProposal(apiPort uint, prop uint) (string, time.Time, error) {
+	url := fmt.Sprintf("http://localhost:%d/cosmos/gov/v1beta1/proposals/%d", apiPort, prop)
+	//url := fmt.Sprintf("https://api.cosmos.network/cosmos/gov/v1beta1/proposals/%d", prop)
```
Comment (gnongs) [inline rpc/cosmos/governance.go:59]: @skonhwang I think it should be removed code. How do you think about?

Comment (skonhwang) [reply rpc/cosmos/governance.go]: @gnongs Really thanks 🙏

**Review by gnongs — APPROVED** [review]
LGTM !

**Review by heejin-github — APPROVED** [review]
LGTM

**Comment by heejin-github** [comment]
governance alarm plugin add to official plugins.

**Pull request merged**